mysql/datadog_checks/mysql/mysql.py

```diff
@@ -401,6 +401,7 @@ def _collect_metrics(self, db, tags):
 
         if is_affirmative(self._config.options.get('table_rows_stats_metrics', False)) and self.userstat_enabled:
             # report size of tables in MiB to Datadog
+            self.log.debug("Collecting Table Row Stats Metrics.")
```

Comment:
added this bc other functions above have it as well e.g. https://github.com/DataDog/integrations-core/pull/12472/files#diff-4ecc52a59650f90cba9b1b1b3a145f8dc3457d133fc52419846c63e62bbe9bd8R383